operations/services.md

```diff
@@ -143,62 +165,50 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `OccupancyStartOffset` | string | required | Offset from the start of the [time unit](../guidelines/time-units.md) which defines the occupancy start of the service; expressed in ISO 8601 duration format. 'Occupancy start' is used for availability and reporting purposes, it implies the time at which the booked resource is considered occupied. |
 | `OccupancyEndOffset` | string | required | Offset from the end of the [time unit](../guidelines/time-units.md) which defines the occupancy end of the service; expressed in ISO 8601 duration format. 'Occupancy end' is used for availability and reporting purposes, it implies the time at which the booked resource is no longer considered occupied. |
 | `TimeUnitPeriod` | [Time unit period](#time-unit-period) | required | The length of time or period represented by a [time unit](../guidelines/time-units.md), for which the service can be booked. |
+| ~~`TimeUnit`~~ | [Time unit period](#time-unit-period) | required | This property is deprecated. |
 
 #### Time unit period
 
-* `Hour`
 * `Day`
 * `Month`
-* ...
```

Comment:
Do we want to remove `...` meaning it can be extended in the future? 🤔

Reply:
I don't know, maybe we can adjust generator and just use the three dots in every enum's description. @jnv what do you think?

Reply:
We have already discussed (maybe with Mike) that this is something we lose in the generated docs.

The problem is that contract-wise it doesn't make sense. Sure, we could put ellipsis on every enum but at that point you can claim that any enum is extensible and ellipsis is just an unnecessary noise. We could add custom attribute to mark enum as extensible, but it would have no value for developers, because contractually we don't provide any safe fallback value.

I think this needs more nuanced approach by, e.g., providing a schema which defines a safe fallback for auto-generated clients.

Reply:
This is a good question. I've always thought the value of the `...` was questionable, especially as we never define what it means in the docs, so it just has an informal meaning. The questions we should ask are (i) do devs writing to our API know what it means? and (ii) does extending the list of enums count as a breaking change or not? Maybe we should have an explanation in the Guidelines to say something like "we reserve the right to extend the range of enum values, so please code your solution accordingly." in which case, every enum counts as `...`.